src/effects/effectchainslot.cpp

```diff
@@ -68,6 +68,12 @@ EffectChainSlot::EffectChainSlot(EffectRack* pRack, const QString& group,
 
     connect(&m_channelStatusMapper, SIGNAL(mapped(const QString&)),
             this, SLOT(slotChannelStatusChanged(const QString&)));
+
+    // It is up to skins and controller mappings to do anything useful with this.
+    m_pControlExpanded = new ControlPushButton(ConfigKey(m_group, "expanded"));
```

Comment:
Right, it's the blessing and curse of having a skin system -- everything is convention. One thing we've learned is that baking UI-concerns into the backend is a form of technical debt. 

If you define these COs in the skin attributes section, I think the outcome would be the same as if we have the CO here, would it not? Skins we ship with would support it, skins we don't ship with won't (same as if we did it in C++).  The difference would be the C++ implementation of the effects system doesn't have to worry about this skin/controller interaction.

We could start a wiki page describing a skin/controller CO interface with some well-defined controls for visual indicators (like focus). Then controllers and skin authors could have a common standard to build against.